asa/ja-jp/Content/Topics/Adv_Server_Access/docs/ports.htm

```diff
@@ -147,40 +149,36 @@
-                                                    <div class="MCBreadcrumbsBox_0 breadcrumbs" role="navigation" aria-label="Breadcrumbs" data-mc-breadcrumbs-divider="  &gt;  " data-mc-breadcrumbs-count="3" data-mc-toc="True">
-                                                    </div>
+                                                    <div class="MCBreadcrumbsBox_0 breadcrumbs" role="navigation" aria-label="ブレッドクラム" data-mc-breadcrumbs-divider=" &gt; " data-mc-breadcrumbs-count="3" data-mc-toc="True">                                                     </div>
                                                 </div>
                                             </div>
                                             <div class="toolbar-main">
                                                 <div class="buttons popup-container clearfix topicToolbarProxy _Skins_okta_toolbar_no_expand mc-component nocontent" style="mc-topic-toolbar-items: ;">
-                                                    <div class="button-group-container-left">                                                         <button class="button needs-pie select-language-button" title="言語の変更">                                                             <div>
-                                                                <div role="img" class="button-icon-wrapper" aria-label="Change language">
-                                                                    <div class="button-icon"> </div>
+                                                    <div class="button-group-container-left">                                                         <button class="button needs-pie select-language-button" title="言語を変更">                                                             <div>
+                                                                <div role="img" class="button-icon-wrapper" aria-label="言語を変更">                                                                     <div class="button-icon"> </div>
                                                                 </div>
                                                             </div>                                                         </button> <button class="button needs-pie previous-topic-button" title="前のトピック" disabled="true">                                                             <div>
-                                                                <div role="img" class="button-icon-wrapper" aria-label="Navigate previous">
-                                                                    <div class="button-icon"> </div>
+                                                                <div role="img" class="button-icon-wrapper" aria-label="前へ移動">                                                                     <div class="button-icon"> </div>
```

Comment:
so this change is the aria-label translated but also removing a line-break at the end (so we have now 2 div elements in the same line). Nothing for you to do there, but if we see a this happening in future translations, we could ask Peng about it